flytekit/clis/sdk_in_container/run.py

```diff
@@ -512,12 +512,19 @@ def _run(*args, **kwargs):
             return
 
         remote = ctx.obj[FLYTE_REMOTE_INSTANCE_KEY]
+        config_file = ctx.obj.get(CTX_CONFIG_FILE)
```

Comment:
just for own understanding, what's the level that config is applied? As in, can you pass the `--config` flag to the `run` command?

Reply:
We still have to use `--config` before the `run`. like `pyflyte --config flyte.yaml run ....`